### PR TITLE
Refactored start.jbpm.org web app to a multi-module project

### PR DESCRIPTION
**Pull request by VaniHaripriya**

### CODE REVIEW

File: .project [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
```
Comment (ricardozanini) [inline .project:2]: This file can be added to `.gitignore`.

File: .settings/org.eclipse.m2e.core.prefs [hunk 2]
```diff
@@ -0,0 +1,4 @@
+activeProfiles=
+eclipse.preferences.version=1
+resolveWorkspaceProjects=true
+version=1
```
Comment (ricardozanini) [inline .settings/org.eclipse.m2e.core.prefs:4]: this one too should be added to `.gitignore`

File: jbpm-bootstrap-kjar/project.imports [hunk 3]
```diff
@@ -0,0 +1,31 @@
+<configuration>
```
Comment (ricardozanini) [inline jbpm-bootstrap-kjar/project.imports:1]: Can be added to `.gitignore`.

File: jbpm-bootstrap-kjar/project.repositories [hunk 4]
```diff
@@ -0,0 +1,20 @@
+<project-repositories>
```
Comment (ricardozanini) [inline jbpm-bootstrap-kjar/project.repositories:1]: Can be added to `.gitignore`.

File: jbpm-bootstrap-kjar/readme.md [hunk 5]
```diff
@@ -0,0 +1,4 @@
+Repository Init Content
```
Comment (ricardozanini) [inline jbpm-bootstrap-kjar/readme.md:1]: Maybe we can add a brief description of this module in this file. Also, please capitalize the name of the file.

**Review by ricardozanini — COMMENTED** [review]
@VaniHaripriya can you please add a `.gitignore` file in the root of this project removing all the Eclipse's files and the target directory? You can use this one as an example: https://github.com/kiegroup/kogito-runtimes/blob/main/.gitignore

**Comment by fjtirado** [comment]
Hi Vani, 
Apart from adding  ignore, since target files were added to control version, I think you need to manually delete them and commit the deletion. 
Probably a faster option will be to close this PR and open a new one with the changes now that you have ignore properly set up

**Comment by VaniHaripriya** [comment]
> Hi Vani,
> Apart from adding ignore, since target files were added to control version, I think you need to manually delete them and commit the deletion.
> Probably a faster option will be to close this PR and open a new one with the changes now that you have ignore properly set up

Hi Francisco,
I deleted the target files manually. I will create a new PR as suggested.

**Comment by VaniHaripriya** [comment]
Creating a new pull request for the same.

**Pull request closed without merging**